File: week3/Eren-Taskiran/704-binary-search.c
int search(int* nums, int numsSize, int target) {
    int l = 0,r=numsSize-1;
    while(l<=r){
        int m = (l+r)/2;
        if(nums[m]==target) return m;
        if(nums[m]>target) r=m-1;
        else l=m+1;
    }
    return -1;
}

File: week3/yusuffenes/704-binary-search.py
def search(self, nums: List[int], target: int) -> int:
    min,flg,max=0,0,len(nums)-1
    while min<=max:
        m=int((min+max)//2)
        if nums[m]==target:
            flg=1
            return m
            break
        elif nums[m]<target:
            min=m+1
        else:
             max=m-1
    if flg == 0:
          return -1